index.php

```diff
@@ -38,10 +57,20 @@ <h3>JUNIOR WEB DEVELOPER</h3>
             <div class="top-sub-heading">
                 <h2>ABOUT ME</h2>
             </div>
-            <p class="main-content-text">I am a junior web developer with a passion for design and agile development. I am currently learning HTML, CSS, Javascript, PHP, and MySQL. I am also familiar with Git and other software environments. I am a certified scrum master and will be applying for my PHP Zend certification after I have graduated from Mayden Academy.</p>
+            <p class="main-content-text"><?php echo $about_me_array["paragraph_1"] ?></p>
+            <div class="article">
+                <img src="<?php echo $about_me_array["image"] ?>" class="image" alt="sea shore">
+                <p class="main-content-text"><?php echo $about_me_array["paragraph_2"] ?></p>
+            </div>
+        </div>
+        <div class="main-content-wrapper">
+            <div class="top-sub-heading" id="C4">
+                <h2>PROJECTS</h2>
+            </div>
+            <p class="main-content-text"><?php echo $projects_array["title"] ?></p>
             <div class="article">
-                <img src="img/a-drop-of-water-osa-water-leaf-158725.jpeg" class="image" alt="sea shore">
-                <p class="main-content-text">In addition to my web-development skills I have an honours degree in marketing and have over ten year's experience in hospitality and customer service. I also possess a level 5 leadership qualification from the Institute of Business Leadership. As far as hobbies and personal time go, I spent a lot of time working on my car, keeping fit, learning German and playing Rugby for the Bath Saracens. I am also currently working part-time at the King William pub in Bath.</p>
+                <img src="<?php echo $projects_array["image"] ?>" class="image" alt="Mayden Academy logo">
```

Comment:
hard coding the alt tag to mayden academy logo means if you change the project the alt will be wrong